src/sentry/db/models/fields/gzippeddict.py

```diff
@@ -32,7 +32,7 @@ def to_python(self, value):
         try:
             if not value:
                 return {}
-            return json.loads(value, skip_trace=True)
```

Comment:
Shouldn't this stay `True` so that we skip creating spans?

Reply:
I removed from `loads()` function directly.

https://github.com/getsentry/sentry/pull/70833/files#diff-5c1440eab19b478087e2a08ff5238d26b56c916cc7baa1b481a9645cab9200abR131